demos/common/python/openvino/model_zoo/model_api/models/model.py

```diff
@@ -18,19 +18,35 @@
 
 
 class WrapperError(RuntimeError):
+    '''Special class for errors occurred in Model API wrappers'''
     def __init__(self, wrapper_name, message):
         super().__init__(f"{wrapper_name}: {message}")
 
 
 class Model:
     '''An abstract model wrapper
 
-    An abstract model wrapper can only load model from the disk.
-    The ``preprocess`` and ``postprocess`` methods should be implemented in a concrete class
+    The abstract model wrapper is free from any executor dependencies.
+    It sets the `ModelAdapter` instance with the model read from disk or other place
```

Comment:
```suggestion
    It sets the `ModelAdapter` instance with the provided model
```
Do we need to specify here where model comes from exactly? (`or other place` sound uncertain)

Reply:
@ivikhrev all suggestions considered